internal/dag/policy.go

```diff
@@ -147,8 +158,8 @@ func timeoutPolicy(tp *projcontour.TimeoutPolicy) *TimeoutPolicy {
 		return nil
 	}
 	return &TimeoutPolicy{
-		ResponseTimeout: annotation.ParseTimeout(tp.Response),
-		IdleTimeout:     annotation.ParseTimeout(tp.Idle),
+		ResponseTimeout: timeout.Parse(tp.Response),
+		IdleTimeout:     timeout.Parse(tp.Idle),
```

Comment:
Now that you have explicit defaults, you can return a default policy if `tp` is nil:
```
if tp == nil {
        return &TimeoutPolicy {
                ResponseTimeout: timeout.DefaultSetting(),
                IdleTimeout: timeout.DefaultSetting(),
        }
}
```

This lets you clean up helpers, per comment below.